translations/en/translation.json

```diff
@@ -22,17 +22,24 @@
     },
     "globalErrors": {
       "reference": {
+        "cannotAccess": "\nError ▶️ \"{{symbol}}\" is used before initialization. Make sure you have initialized the variable before using it.\n\n{{location}}\n\nFor more: {{url}}",
```

Comment:
I would suggest: `"{{symbol}}\" is used before being initialized. Make sure you have initialized the variable before using it.\n\n{{location}}\n\nFor more: {{url}}"`

Reply:
I know in JS variables are automatically initialized when they are declared but I wonder if it would be more helpful to novices if we talked about declaration instead of initialization. @Aloneduckling @outofambit @Qianqianye  what do you think?

Reply:
Yeah talking about declaration also makes sense. 
I will wait for the input from all of you and then make the changes.

Reply:
@lm-n Yeah I agree on focusing on declaration instead of initialization. The two concepts are pretty nuanced for beginners so I'd vote we pick one to focus on. (Generally, you can't initialize something that hasn't been declared plus what you said about JS variables getting initialized as soon as they are declared even if you omit an explicit value).